Classes/Controller/FrontEndEditorController.php

```diff
@@ -41,4 +49,33 @@ private function getUidOfLoggedInUser(): int
     {
         return $this->context->getPropertyFromAspect('frontend.user', 'id');
     }
+
+    public function editAction(Tea $tea): ResponseInterface
+    {
+        $this->checkIfUserIsOwner($tea);
+
+        $this->view->assign('tea', $tea);
+
+        return $this->htmlResponse();
+    }
+
+    /**
+     * @throws \RuntimeException
+     */
+    private function checkIfUserIsOwner(Tea $tea): void
+    {
+        if ($tea->getOwnerUid() !== $this->getUidOfLoggedInUser()) {
+            throw new \RuntimeException('You do not have the permissions to edit this tea.', 1687363749);
```

Comment:
Note: I'd like to change this to a proper alert in the FE and a "forbidden" HTTP header in a later change (and would like to keep that functionality out of this PR chain here).